nb.lproj/Localizable.strings

```diff
@@ -0,0 +1,161 @@
+/* 
+  Localizable.strings
+  MobileRadio
+
+  Created by Christoffer Tønnessen on 2020-05-09.
+  Copyright © 2019 High Caffeine Content. All rights reserved.
+*/
+
+"SHORTCUT_PLAY" = "Spill av";
+"SHORTCUT_PAUSE" = "Pause";
+
+"TOOLBAR_NOTHING_PLAYING" = "Ingenting spilles";
+"TOOLBAR_STREAMING_RADIO" = "Internettradio";
+
+"TOOLBAR_PLAYBUTTON_ACCESSIBILITY_LABEL" = "Spill av/Pause";
+"TOOLBAR_PLAYBUTTON_TOOLTIP" = "Spill av/Pause";
+"TOOLBAR_MUTEBUTTON_TOOLTIP" = "Lydløs";
+"TOOLBAR_VOLUMESLIDER_ACCESSIBILITY_LABEL" = "Volum";
+
+"SIDEBAR_TITLE_LIBRARY" = "Bibliotek";
+"SIDEBAR_TITLE_STATIONS" = "Samling";
+"SIDEBAR_TITLE_RADIO" = "Radio";
+
+"SIDEBAR_CATEGORY_ALL_STATIONS" = "Alle stasjoner";
+"SIDEBAR_CATEGORY_RECENTS" = "Nylige";
+
+"LIST_HEADER_STATION" = "Stasjon";
+
+"MENU_NEW" = "Ny";
+"MENU_NEW_STATION" = "Stasjon…";
+"MENU_NEW_GROUP" = "Samling…";
+
+"DISCOVERABILITY_HUD_NEW_STATION" = "Ny stasjon";
+"DISCOVERABILITY_HUD_NEW_GROUP" = "Ny samling";
+
+"MENU_VIEW_AS_GRID" = "som rutenett";
+"MENU_VIEW_AS_LIST" = "som liste";
+
+"DISCOVERABILITY_HUD_VIEW_AS_GRID" = "Se som rutenett";
+"DISCOVERABILITY_HUD_VIEW_AS_LIST" = "Se som liste";
+
+"MENU_VIEW_SHOW_SIDEBAR" = "Vis sidemeny";
+"MENU_VIEW_HIDE_SIDEBAR" = "Gjem sidemeny";
+
+"MENU_HELP_GETSUPPORT" = "Få hjelp på Twitter…";
+"MENU_HELP_SHOW_WELCOME_SCREEN" = "Vis velkomstskjermen";
+
+"PLAYBACK_ERROR_STALL" = "Kobling til stasjonen er brutt.";
+
+"SEARCH_PLACEHOLDER" = "Søk";
+
+"ALERT_OK" = "OK";
+"ALERT_ADD" = "Legg til";
+"ALERT_CANCEL" = "Avbryt";
+"ALERT_DELETE" = "Slett";
+
+"ADD_STATION_WINDOW_TITLE" = "Legg til ny stasjon";
+"EDIT_STATION_WINDOW_TITLE" = "Rediger stasjon";
+
+"EDIT_STATION_WINDOW_NAME_PLACEHOLDER" = "Navn";
+"EDIT_STATION_WINDOW_URL_PLACEHOLDER" = "http://adresse/til/stasjonen";
+"EDIT_STATION_WINDOW_IMAGE_ACCESSIBILITY_LABEL" = "Velg bilde";
+
+"ADD_COLLECTION_WINDOW_TITLE" = "Legg til ny samling;
```

Comment:
Missing a trailing `"` here, if you could add that would be great. 🙏

Reply:
Fixed!